apps/studio/lib/ai/tools/schema-tools.ts

```diff
@@ -0,0 +1,49 @@
+import { tool } from 'ai'
+import { getDatabasePolicies } from 'data/database-policies/database-policies-query'
+import { z } from 'zod'
+
+export const getSchemaTools = ({
+  projectRef,
+  connectionString,
+  authorization,
+}: {
+  projectRef: string
+  connectionString: string
+  authorization?: string
+}) => ({
+  list_policies: tool({
```

Comment:
just out of curiosity - wondering why this doesn't live in MCP tools like list_tables, list_extensions, etc